networking/libsnnet/docker_plugin.go

```diff
@@ -171,7 +171,7 @@ func (d *DockerEpMap) NewTable() {
 
 //Name provides the name of the map
 func (d *DockerEpMap) Name() string {
-	return tableNetworkMap
+	return tableEndPointMap
```

Comment:
wow.. that was quite a big bug at my end. I wonder how things worked in spite of this bug.